protocol/README.md

```diff
@@ -25,147 +25,148 @@ Every TBDex message contains the following fields:
-| `paymentTypes`   | JSON Object    | N        |  A normalized object describing the type of payments you need quotes for. When absent, the PFI is free to include any payment types.                 |
+| `kind` | enum    | Y        | Type of payment instrument (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`)|
+| `fee` | object    | N        | Optional fee associated with using this kind of payment instrument.|
 
+PFI -> Alice: "Here's what I can offer if you want to buy BTC with USD, and here are the constraints of my offer, in terms of how much you can buy, what credentials I need from you, and what payment instruments you can use to pay me, and what payment instruments I can use to pay you."
 ```json
 {
-  "id" : "1234",
-  "have": "AUD",
-  "want": "Cheese",
-  "side": "have",
-  "size": 100
+  "pair": "BTC_USD",
+  "unitPrice": 27000.00,
+  "baseFee": 1.00,
+  "min": 10.00,
+  "max": 100.00,
+  "presentationRequest": { /* Presentation Request for KYC VC */ },
+  "payinInstruments": [{
+      "kind": "DEBIT_CARD",
+      "fee": {
+        "flatFee": 1.00
+      }
+  }],
+  "payoutInstruments": [{
+      "kind": "BTC_ADDRESS"
+  }]
 }
 ```
-I have 100 AUD, and I want Cheese
+## `RequestForQuote`
+
+| field            | data type | required | description                                                                                          |
+| ---------------- | --------- | -------- | ---------------------------------------------------------------------------------------------------- |
+| `rfqId` | string    | Y        | A unique identifier for this Request for Quote. |
+| `pair` | string    | Y        | The currency pair being offered, in the format of `basecurrency_countercurrency`.|
+| `amount` | string    | Y        | Amount of counter currency you want to spend in order to receive base currency|
+| `verifiablePresentation` | VerifiablePresentation    | Y        | VerifiablePresentation that meets the specification per PresentationRequest in the Offering |
+| `payinInstrument`   | PaymentInstrument       | Y        | Specify which payment instrument to send counter currency. |
+| `payoutInstrument`   | PaymentInstrument       | Y        | Specify which payment instrument to receive base currency. |
+
+Alice -> PFI: "OK, that offering looks good. I want a Quote against that Offering, and here is how much counter currency I want to trade for your base currency. Here's the payment instrument I intend to pay you with, and here's the payment instrument I expect you to pay me in."
 
 ```json
 {
-  "id": "1234",
-  "have": "AUD",
-  "want": "Cheese",
-  "side": "want",
-  "size": 100
+  "rfqId": "1",
+  "pair": "BTC_USD",
+  "amount": 10.00,
+  "verifiablePresentation": { /* Alice's KYC VC in Verifiable Presentation format */ },
+  "payinInstrument": {
+      "kind": "DEBIT_CARD"
+  },
+  "payoutInstrument": {
+      "kind": "BTC_ADDRESS"
+  }
```

Comment:
The `kind` here is of type `BTC_ADDRESS`, but where would Alice provide their actual BTC address to the PFI? Would that be within this `payoutInstrument` object, or some other area?

Reply:
so, during the `RFQ` step, Alice actually would not present her actual BTC address to the PFI. this is done during the `Order` step: 

```
{
    "orderId": 32432,
    "quoteId": 2,
    "paymentVerifiablePresentation": { /* Alice's Debit Card and Bitcoin VC in Verifiable Presentation format  */}
}
```

the `paymentVerifiablePresentation` is what's submitted to the PFI. it contains 2 x VCs, one for pay-in (her debit card) and another for pay-out (her actual BTC address). 

the general idea is that 
1. Alice says during the RFQ step "here's how i intend to pay", then
2. PFI's Quote says "ok, if you're gonna pay with those, here's how to shape your payment details whenever you're ready to send that in"
3. Alice's Order says "here's my payment details shaped how you want them"

does that make sense?